server/database/queries/getCohortData.js

```diff
@@ -0,0 +1,13 @@
+const db_connection = require("../config/db_connection");
+const getCohortData = (cohortName) => {
```

Comment:
@Hussein-shahatet97 Do cohort rows have an ID?  It might be safer to have this endpoint accept an ID instead of a name.  Names of things sometimes change after they have been created, so it's best to use IDs in URLs so that those URLs don't become "broken links" if a row's name changes later.